packages/database/src/api/Database.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2017 Google Inc.
+ * Copyright 2017 Google LLC
```

Comment:
Per go/copyright, we are not supposed to update copyright notices in old files.

Reply:
This is the PR that causes this update: https://github.com/firebase/firebase-js-sdk/pull/2665

From the description it sounds like it is intentional that it should update changed files and I think that's what go/copyright says (retain oldest year, change Inc. to LLC)